config/bdw.toml

```diff
@@ -0,0 +1,15 @@
+version = [1, 0]  # use simple file write
```

Comment:
Intel legacy platforms are no longer in main branch - i.e. cavs2.5 and above is for main.